### PR TITLE
Default to yes in yes/no prompt logic.

### PR DESCRIPTION
**Pull request by ser-io**

### CODE REVIEW

File: pkg/cli/cli.go [hunk 1]
```diff
@@ -326,9 +326,10 @@ func PromptSelectionFromMap[K comparable, T any](c *command, choices map[K]T, to
 
 func PromptYesOrNo(out *os.File, in *os.File, text string) (bool, error) {
 	fmt.Fprint(out, text+" (y/N): ")
```
Comment (jemoreira) [inline pkg/cli/cli.go:328]: `(Y/n)`

Comment (ser-io) [reply pkg/cli/cli.go]: Done. Also change the logic to accept lower and upper case values for both: `y` and `n`.

File: pkg/cli/cli.go [hunk 2]
```diff
@@ -325,16 +325,18 @@ func PromptSelectionFromMap[K comparable, T any](c *command, choices map[K]T, to
...
-	return yN == "y", nil
+	fmt.Fprint(out, text+" (Y/n): ")
+	yn := "Y"
+	_, err := fmt.Fscanln(os.Stdin, &yn)
+	// Using the error text for comparison as there's no specific error type to compare against.
+	if err != nil && err.Error() != "unexpected newline" {
+		return false, fmt.Errorf("failed to read (Y/n) choice: %w", err)
+	}
+	ynLo := strings.ToLower(yn)
+	if ynLo != "y" && ynLo != "n" {
```
Comment (jemoreira) [inline pkg/cli/cli.go:336]: nit: the user could write "yes" or "no", include those here and in the return compare `ynLo[0] == 'y'`

**Review by jemoreira — APPROVED** [review]

File: pkg/cli/cli.go (hunk 2 above)
Comment (ser-io) [reply pkg/cli/cli.go]: Done.

**Review by jemoreira — APPROVED** [review]

**Pull request merged**